src/urllib3/exceptions.py

```diff
@@ -209,14 +227,15 @@ class DependencyWarning(HTTPWarning):
 
 
 class ResponseNotChunked(ProtocolError, ValueError):
-    "Response needs to be chunked in order to read it as chunks."
+    """Response needs to be chunked in order to read it as chunks."""
+
     pass
 
 
 class BodyNotHttplibCompatible(HTTPError):
-    """
-    Body should be httplib.HTTPResponse like (have an fp attribute which
-    returns raw chunks) for read_chunked().
+    """ "
```

Comment:
Looks like you have an extra quote here

Reply:
Should we explain why this exception still as httplib in the name?

Reply:
eh, I'm pretty sure this is a pretty rare one and the docstring explains it enough?